nixos/modules/services/x11/desktop-managers/plasma5.nix

```
@@ -387,7 +387,8 @@ in
         ++ lib.optional config.services.colord.enable pkgs.colord-kde
         ++ lib.optional config.services.hardware.bolt.enable pkgs.plasma5Packages.plasma-thunderbolt
         ++ lib.optionals config.services.samba.enable [ kdenetwork-filesharing pkgs.samba ]
-        ++ lib.optional config.services.xserver.wacom.enable pkgs.wacomtablet;
+        ++ lib.optional config.services.xserver.wacom.enable pkgs.wacomtablet
+        ++ lib.optional config.services.flatpak.enable pkgs.flatpak-kcm;
```

`flatpak-kcm` isn't in the root pkgs

```suggestion
        ++ lib.optional config.services.flatpak.enable pkgs.plasma5Packages.flatpak-kcm;
```

---

Nice catch, fixed.